docs/charter.md

```diff
@@ -1,28 +1,70 @@
 ICU4X Charter
 ==============
 
-ICU4X is a new project whose objective is to solve the needs of clients who wish to provide client-side i18n for their products in resource-constrained environments.
+The ICU4X project designs and develops a set of modular internationalization components suitable for use on the client-side and in resource-constrained environments.
```

Comment:
"client-side", with a hyphen, is an adjective.  You should say either "for use on the client side and in resource-constrained environments" or "for use in client-side or resource-constrained environments".

(I'm being extra picky since this is the first sentence of the charter)